content/en/docs/concepts/security/index.md

```diff
@@ -630,333 +631,277 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
-- **which conditions** refers to the `conditions` section you can specify with
-  the [Istio attributes](/docs/reference/config/policy-and-telemetry/attribute-vocabulary/)
-  in either `ServiceRole` or `ServiceRoleBinding`.
+- **target** refers to the `selector` section in the `AuthorizationPolicy`.
+- **who** refers to the `from` section in the `rule` of the `AuthorizationPolicy`.
+- **what** refers to the `to` section in the `rule` of the `AuthorizationPolicy`.
+- **conditions** refers to the `when` section in the `rule` of the `AuthorizationPolicy`.
 
-#### `ServiceRole`
-
-A `ServiceRole` specification includes a list of `rules`, AKA permissions.
 Each rule has the following standard fields:
 
-- **`services`**: A list of service names. You can set the value to `*` to
-  include all services in the specified namespace.
-
-- **`methods`**: A list of HTTP methods. You can set the value to `*` to include all
-  HTTP methods. This field should not be set for TCP and gRPC services.
-
-- **`paths`**: HTTP paths or gRPC methods. The gRPC methods must be in the
-   form of `/packageName.serviceName/methodName` and are case sensitive.
+- **`from`**: A list of sources.
+- **`to`**: A list of operations.
+- **`when`**: A list of custom conditions.
 
-A `ServiceRole` specification only applies to the namespace specified in the
-`metadata` section. A rule requires the `services` field and the other fields are optional.
-If you do not specify a field or if you set its value to `*`, Istio applies the field to all instances.
-
-The example below shows a simple role: `service-admin`, which has full access
-to all services in the `default` namespace.
+The following example shows an `AuthorizationPolicy` that allows two sources
+(service account `cluster.local/ns/default/sa/sleep` and namespace `dev`) to access the
+workloads with labels `app: httpbin` and `version: v1` in namespace foo when the request
+is sent with a valid JWT token.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRole
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: service-admin
-  namespace: default
+ name: httpbin
+ namespace: foo
 spec:
-  rules:
-  - services: ["*"]
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   - source:
+       namespaces: ["dev"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.auth.claims[iss]
+     values: ["https://accounts.google.com"]
 {{< /text >}}
 
-Here is another role: `products-viewer`, which has read, `"GET"` and `"HEAD"`,
-access to the service `products.default.svc.cluster.local` in the `default`
-namespace.
+#### Policy Target
+
+Policy scope (target) is determined by `metadata/namespace` and an optional `selector`.
+
+The `metadata/namespace` tells which namespace the policy applies. If set to the
```

Comment:
The `metadata/namespace` tells which namespace the policy applies to. If set to the